using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -4,8 +4,160 @@
 
 The following events are handled by the Identity extension:
 
+### Analytics Identity Response
```

Comment:
We should move these events after the Identity request Identity and Generic Identity request events.